packages/mui-base/src/Portal/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './Portal';
```

Comment:
I think you should re-export `PortalProps` from the `index.ts`.
That way you can revert the changes you made to import it across the codebase.

Reply:
I am on it. Thanks.